### DISCUSSION TITLE
sass url() not working in production / staging

### DISCUSSION THREAD

**Discussion by chmich** [ElMassimo/vite_ruby#311] [Q&A] [answered]
Hi, there, after swisscowing (not: googling) two days i came to the point to ask you:

What do i have missconfigured on my vite?

`frontend/stylesheets/components/global-select.scss`

```
background-image: url("./../../images/fa-solid/magnifying-glass-lightgray.svg");
```

is working fine under development, on my local machine, but not after deploying to staging / production. Precompilation while deploying, by capistrano, is running well on that machines, all styles are working, except `url()`.

On Staging, the Firefox Web Inspector says:

![grafik](https://user-images.githubusercontent.com/31048265/204822887-1358e597-3a04-44ea-a928-69268ee9fc85.png)

(could not load graphic)

How do you insert images?

**Thanks,
Christian**

**Comment by chmich** [comment, marked as answer]
found it:

`background-image: url("@/images/fa-solid/magnifying-glass-lightgray.svg");`

works.
it seems that the `@` is referencing to `app/frontend/` and on that way its working on all environments.
And, i did not have to configure some aliases like [Stackoverflow](https://stackoverflow.com/questions/67744210/how-to-use-asset-urls-in-style-binding-with-vite) or [plainenglish.io](https://javascript.plainenglish.io/how-to-set-up-path-resolving-in-vite-ad284e0d9eae) saying. it seems to be be configured by default.

**Reply by ElMassimo** [reply]
That's correct, Vite Ruby configures [`~` and `@` import aliases](https://vite-ruby.netlify.app/guide/development.html#import-aliases-%F0%9F%91%89) out of the box.

The [_Migration_ section](https://vite-ruby.netlify.app/guide/migration.html#assets-%F0%9F%8E%A8) in the docs mentions how to reference assets in the project.
